src/resources/postcss/views/_day.pcss

```diff
@@ -5,4 +5,86 @@
  *
  */
 
-.tribe-events {}
+.tribe-events {
+
+	.tribe-events-calendar-day__event {
+		display: flex;
+		flex-direction: column;
+		margin-bottom: var(--spacer-4);
+		margin-top: var(--spacer-7);
+
+		@media (--viewport-medium) {
+			flex-direction: row-reverse;
+			justify-content: flex-end;
+			margin-bottom: var(--spacer-6);
+			margin-top: var(--spacer-9);
+		}
+
+		&:last-child {
+			margin-bottom: var(--spacer-7);
+
+			@media (--viewport-medium) {
+				margin-bottom: var(--spacer-9);
+			}
+		}
+	}
+
+	.tribe-events-calendar-day__event-featured-image-wrapper {
+		flex: none;
+		margin-bottom: var(--spacer-4);
+
+		@media (--viewport-medium) {
+			margin-bottom: 0;
+			width: calc(3 * var(--grid-width-1-of-9));
+		}
+	}
+
+	.tribe-events-calendar-day__event-featured-image {
+		padding-bottom: 56.25%;
+	}
+
+	.tribe-events-calendar-day__event-details {
+		flex: none;
+
+		@media (--viewport-medium) {
+			margin-left: var(--grid-width-1-of-9);
+			width: calc(5 * var(--grid-width-1-of-9));
+		}
+	}
+
+	.tribe-events-calendar-day__event-datetime-wrapper {
+		align-items: center;
+		display: flex;
+		margin-bottom: var(--spacer-1);
+		position: relative;
+
+		@media (--viewport-medium) {
+			margin-bottom: var(--spacer-2);
+		}
+	}
+
+	.tribe-events-calendar-day__event-datetime-featured-icon {
+		margin-left: 10px;
+	}
+
+	.tribe-events-calendar-day__event-title {
+		margin-bottom: var(--spacer-1);
+
+		@media (--viewport-medium) {
+			margin-bottom: var(--spacer-2);
+		}
+	}
+
+	.tribe-events-calendar-day__event-title-link {
+		@mixin anchor-default;
```

Comment:
actually i forgot we had `tribe-common-anchor` class defined. let's use that on the event title link instead.